mix.exs

```diff
@@ -60,9 +61,14 @@ defmodule PostHog.MixProject do
       {:nimble_options, "~> 1.1"},
       {:req, "~> 0.5.10"},
       {:logger_json, "~> 7.0"},
+      {:nimble_ownership, "~> 1.0"},
       # Development tools
       {:ex_doc, "~> 0.37", only: :dev, runtime: false},
-      {:logger_handler_kit, "~> 0.3", only: :test},
+      # {:logger_handler_kit, "~> 0.3", only: :test},
+      {:logger_handler_kit,
```

Comment:
Would rather we only merged this once you've released a new `logger_handler_kit` version, is that coming?

Reply:
of course! Just wanted to get this reviewed first. A bit of a chicken and egg problem 😄